packages/utils/src/utils/index.js

```diff
@@ -356,6 +356,14 @@ export function getGenerateNamePrefixForRerun(name) {
   return `${root}${rerunIdentifier}`;
 }
 
+export function getGenerateNamePrefixForNewRun(name) {
+  let generateName = name;
+  if (!name.endsWith('-')) {
+    generateName = `${name}-`;
+  }
```

Comment:
This function is only called if `generateName` is not provided, and a valid `name` cannot end with `-` so this function could be simplified to just:

```js
export function getGenerateNamePrefixForNewRun(name) {
  return `${name}-`;
}
```

Instead of having a new function for this I'd suggest just doing it inline where it's used since it's only used in one place…